色々変更(要確認)

kazu0617 opened the pull request:
主にキャリブレータを中心に表現を追加・修正。
よければ通す前に確認してもらえると幸いです。

kazu0617 committed:
色々変更(要確認)

kazu0617 committed:
文章調整2
インポーターに原文で行われている改行を追加
匿名→未登録に変更
オーバーライドを日本語化
前回編集時のTypoを修正
実際に使用した際のずれを修正(スクリーンショット)

Aesc3625 commented on ja.json:
```diff
@@ -225,8 +225,8 @@
-        "World.Detail.Users": "ユーザー",
-        "World.Detail.Users.Count": "{active}{away, select, 0 {} other { (+{away} away)}} / {max}",
+        "World.Detail.Users": "ユーザー人数(人)",
+        "World.Detail.Users.Count": "{active}{away, select, 0 {} other { (+{away} 人離席中)}} / {max}",
```
ユーザーでもう人とわかるので"人数"はまだしも、"(人)"は余計です
    "World.Detail.Users": "ユーザー数",
    "World.Detail.Users.Count": "{active}{away, select, 0 {} other { (離席中+{away})}} / {max}",
自分ならこうするかな

kazu0617 replied:
その通りだと思います。変更します。

Aesc3625 commented on ja.json:
```diff
@@ -417,16 +417,16 @@
-        "Session.Permission.PermissionOverrideCount": "権限オーバーライド: {n,select, -1 {---} other {{n}}}",
-        "Session.Permission.ClearOverrides": "ユーザーオーバーライドを消去",
+        "Session.Permission.PermissionOverrideCount": "個別の権限設定を行ったユーザー数(人): {n,select, -1 {---} other {{n}}}",
+        "Session.Permission.ClearOverrides": "ユーザーごとに設定した個別の権限設定を消去",
```
これも「ユーザーごと」と「個別」で意味が重なってるので
        "Session.Permission.PermissionOverrideCount": "個別の権限設定を行ったユーザー数: {n,select, -1 {---} other {{n}}}",
        "Session.Permission.ClearOverrides": "ユーザーごとに設定した権限設定を消去",

kazu0617 replied:
変更します。付け加えて、---がわかりづらかったので「表示できません」という文言にするコミットを入れます。

Aesc3625 commented on ja.json:
```diff
@@ -417,16 +417,16 @@
-        "Permissions.NotAllowedToSwapAvatar": "ここではカスタムアバターの仕様は許可されていません。",
+        "Permissions.NotAllowedToSpawn": "このセッションで物をスポーンすることは許可されていません。",
+        "Permissions.NotAllowedToSave": "このセッションでアイテムの保存は許可されていません。",
+        "Permissions.NotAllowedToSwapAvatar": "このセッションでカスタムアバターに着替えることは許可されていません。",
```
これは警告文でなので、どちらかというと短く簡潔にしたほうがいいとお思います
"ここで物を出すことはできません。",
"ここでアイテムは保存できません。",
"ここでカスタムアバターは使用できません。"

kazu0617 replied:
変更します。ありがとうございます。

Aesc3625 commented on ja.json:
```diff
@@ -656,45 +656,45 @@
-        "Importer.Model.AutoHumanoid": "ヒューマノイドの​身長に自動設定",
+        "Importer.Model.ModelKindPrompt": "どんなモデルですか?",
+        "Importer.Model.RegularKind": "一般的な​モデル",
+        "Importer.Model.SeparableKind": "メッシュごとに​スナップできるように​分離するべき​モデル",
```
"メッシュごとに​スナップできるように​分離するべき​モデル"
意味がわかりづらくなってるので、
"メッシュごとに​スナップするように分離"とか
"メッシュを分離して、スナップ可能にする"かな？

kazu0617 replied:
これについては長くすることで「通常はこっちを使わなくていいんだな」感を出したくて長くしたところがあります。
`"Importer.Model.SeparableKind": "メッシュごとに​分離しスナップ​するように変更<size=50%>ジグソーパズルなど</size>",`
だとどうでしょう？

Aesc3625 commented on ja.json:
```diff
@@ -790,51 +790,51 @@
         "NewWorld.DefaultName": "{name} World",
 
-        "FullBody.Title": "キャリブレート",
+        "FullBody.Title": "キャリブレート | 身長",
```
キャリブレーターについてはまた相談させてください。直接話しながらが早そう。
（詳しく書こうとして文章が長くなると、逆に読まれなくなっちゃうと懸念してます）

kazu0617 replied:
一通り長くなった部分を分割したり端折ったりする変更を加えましたので確認をお願いします。

kazu0617 committed:
文章調整
ユーザー人数→ユーザー数
権限オーバーライド
Permissions.Not**
Importer.Model.ModelKind回り
キャリブレータに関する文章
